src/package_io/parse_input.star

```diff
@@ -253,40 +258,24 @@ def enrich_mev_extra_params(parsed_arguments_dict, mev_prefix, mev_port, mev_typ
 
 	num_participants = len(parsed_arguments_dict["participants"])
 
-	if mev_type == "full":
-
-		mev_url = "http://{0}{1}:{2}".format(mev_prefix, num_participants, mev_port)
-
-		mev_participant = {
-			"el_client_type": "geth",
-			# TODO replace with actual when flashbots/builder is published
-			"el_client_image": "ethpandaops/flashbots-builder:main",
-			"el_client_log_level":    "",
-			"cl_client_type":         "lighthouse",
-			# THIS overrides the beacon image
-			"cl_client_image":        "sigp/lighthouse",
-			"cl_client_log_level":    "",
-			"beacon_extra_params":    [
-				"--builder={0}".format(mev_url),
-				"--always-prepare-payload",
-				"--prepare-payload-lookahead",
-				"12000"
-				],
-			# TODO(maybe) make parts of this more passable like the mev-relay-endpoint & forks
-			"el_extra_params": [
-				"--builder",
-				"--builder.remote_relay_endpoint=http://mev-relay-api:9062",
-				"--builder.beacon_endpoints=http://cl-{0}-lighthouse-geth:4000".format(num_participants+1),
-				"--builder.bellatrix_fork_version=0x30000038",
-				"--builder.genesis_fork_version=0x10000038",
-				"--builder.genesis_validators_root={0}".format(package_io.GENESIS_VALIDATORS_ROOT_PLACEHOLDER),
-				"--miner.extradata=\"Illuminate Dmocratize Dstribute\"",
-				"--builder.algotype=greedy"
-				] + parsed_arguments_dict["mev_params"]["mev_builder_extra_args"],
-			"validator_extra_params": ["--builder-proposals"],
-			"builder_network_params": None
-		}
-
-		parsed_arguments_dict["participants"].append(mev_participant)
+	mev_url = "http://{0}{1}:{2}".format(mev_prefix, num_participants, mev_port)
```

Comment:
the if condition is necessary! we only do this when the mev is of type full not mock